packages/twenty-server/src/modules/messaging/message-import-manager/drivers/gmail/messaging-gmail-driver.module.ts

```diff
@@ -42,11 +43,13 @@ import { OAuth2ClientProvider } from 'src/modules/messaging/message-import-manag
     GmailFetchByBatchService,
     GmailGetMessagesService,
     GmailGetMessageListService,
+    GmailGetAllFoldersService,
```

Comment:
seeing that this is under `message-import-manager` driver, I wonder if 
- alternative 1: message-folder-sync should actually be under `message-import-manager `
- alternative 2: message-folder-sync is next to `message-import-manager` and  if we should create another `drivers` folder under it
- alternative 3: keep as you have done which is not perfect isolation but is practical (drivers under message-import-manager only and message-sync-manager next to message-import-manager)

I'm fine with all alternatives